src/openms/include/OpenMS/METADATA/SpectrumMetaDataLookup.h

```diff
@@ -283,6 +298,8 @@ namespace OpenMS
     /// Assignment operator (not implemented)
     SpectrumMetaDataLookup& operator=(const SpectrumMetaDataLookup&);
 
+    /// adds Ion matches to identifcations
+    static void addIonMatches_(PeptideIdentification& pi, const MSSpectrum<Peak1D> &spec);
```

Comment:
I don't say that you can't use this class for lookup. I just say this class is a bad place to put the ion annotation code. But as I said - lets discuss this offline.